README.md

```diff
@@ -35,6 +35,12 @@ the review process quicker and more streamlined.
 
 ## Install
 
+To use The Lounge you must have [Node.js](https://nodejs.org/en/download/) installed.
+The oldest Node.js version we support is 4.2.0.
```

Comment:
Yeah, I meant <= 4.2 and > 4.0. Why supporting 4.2 as oldest version and not 4.0 is a better question, actually.